project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -72,7 +71,7 @@
   private static final String PLACEID_2 = "ChIJ02qnq0KuEmsRHUJF4zo1x4I";
 
   /** Valid Place objects. */
-  private static final Place PLACE_1 =
+  private static final Place.Builder PLACE_1_BUILDER =
```

Comment:
If you wish to use a Builder object, use a method and not a static final (createValidPlaceBuilder()), or better yet: createValidPlace(String... cuisines) - no builder, use the varargs whenever you need to.

Reply:
Much better :)